config/helpers.php

```diff
@@ -202,12 +193,8 @@ function gist(string $url, string|null $file = null): string
 	/**
 	 * Redirects to the given Urls
 	 * Urls can be relative or absolute.
-	 *
-	 * @param string $url
-	 * @param int $code
-	 * @return void
 	 */
-	function go(string $url = '/', int $code = 302)
+	function go(string $url = '/', int $code = 302): void
```

Comment:
Could use a `@todo` comment to change to `: never` once support for PHP 8.0 is dropped.